### PR TITLE
Make 0.30 LTS

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: software_versioned_docs/version-0.25/release-notes.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ sidebar_label: 'Release Notes'
 id: release-notes
 ---
 
-## Overview
+This document contains all release notes for Astronomer Software 0.25.
 
-Astronomer v0.25 is the latest available Long-Term Support (LTS) version for Astronomer Software. For more information about Software release channels, read [Release and Lifecycle Policies](release-lifecycle-policy.md).
+0.30 is the latest long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, read [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, read [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.25/release-notes.md:9]:
```suggestion
0.30 is the latest long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, read [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
```

File: software_versioned_docs/version-0.29/release-notes.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ description: Astronomer Software release notes.
 
 This document includes all release notes for Astronomer Software version 0.29.
 
-0.29 is the latest stable version of Astronomer Software, while 0.28 remains  the latest LTS long-term support (LTS) version of Astronomer Software. To upgrade to 0.29, read [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, read [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
+0.30 is the latest long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, read [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, read [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.29/release-notes.md:13]:
```suggestion
0.30 is the latest long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To review release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
```

File: software_versioned_docs/version-0.28/release-notes.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ description: Astronomer Software release notes.
 
 This document includes all release notes for Astronomer Software v0.28.
 
-This is the latest LTS long-term support (LTS) version of Astronomer Software. To upgrade to Astronomer v0.28 from v0.25+, see [Upgrade to v0.28](upgrade-astronomer.md). For more information about Software release channels, read [Release and Lifecycle Policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI Release Notes](cli-release-notes.md).
+0.30 is the latest long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, read [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, read [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.28/release-notes.md:14]:
```suggestion
0.30 is the latest long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To review release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
```

File: software_versioned_docs/version-0.25/release-notes.md [hunk 4]
```diff
@@ -4,11 +4,11 @@ sidebar_label: 'Release Notes'
...
 
-If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronomer v0.25](upgrade-astronomer.md). For instructions on how to upgrade to a patch version within the Astronomer v0.25 series, refer to [Upgrade to a Patch Version of Astronomer Software](upgrade-astronomer.md).
+If you're looking to upgrade to Astronomer v0.30, refer to [Upgrade to Astronomer v0.25](upgrade-astronomer.md). For instructions on how to upgrade to a patch version within the Astronomer v0.25 series, refer to [Upgrade to a Patch Version of Astronomer Software](upgrade-astronomer.md).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.25/release-notes.md:11]:
```suggestion
If you're upgrading to Astronomer v0.30, see [Upgrade to Astronomer v0.25](upgrade-astronomer.md). For instructions on how to upgrade to a patch version within the Astronomer v0.25 series, see [Upgrade to a Patch Version of Astronomer Software](upgrade-astronomer.md).
```

File: software/release-notes.md [hunk 5]
```diff
@@ -45,4 +45,4 @@ As part of this change, you can now configure `jwt.authDuration` in your [Housto
 - Fixed an issue where Workspace-level service accounts could view Deployment information from Deployments outside of their Workspace.
 - Fixed an issue where updating the role of a user in a Team using the Astro CLI would not throw an error as expected.
 - Fixed an issue where JSON web tokens persisted after a user logged out if `idpGroupsRefreshEnabled` was set to `false`.
-- Fixed an issue where users who authenticated with Google Direct were automatically logged out of the Software UI after 1 hour.
\ No newline at end of file
+- Fixed an issue where users who authenticated with Google Direct were automatically logged out of the Software UI after 1 hour.
```
Comment (kwcanuck) [inline software/release-notes.md:48]:
```suggestion
- Users authenticating with Google Direct are no longer automatically logged out of Astronomer Software after 1 hour.
```

**Review by kwcanuck — APPROVED** [review]

File: software/release-lifecycle-policy.md [hunk 6]
```diff
@@ -92,13 +92,14 @@ The following tables contain the exact lifecycle for each published version of A
 | 0.25             | May 11, 2021    | December 2022*         |
 | 0.28             | Feb 15, 2022    | February 2023           |
 | 0.29             | June 1, 2022    | December 2022           |
-| 0.30             | August 23, 2022 | February 2022           |
+| 0.30             | August 29, 2022 | February 2022           |
```
Comment (cmarteepants) [inline software/release-lifecycle-policy.md:95]: > February 2022
Should be August 2023

**Review by cmarteepants — CHANGES_REQUESTED** [review]

File: software/release-lifecycle-policy.md (hunk 6 above)
Comment (jwitz) [inline software/release-lifecycle-policy.md:95]:
```suggestion
| 0.30             | August 29, 2022 | February 2023           |
```

File: software_versioned_docs/version-0.25/release-lifecycle-policy.md [hunk 7]
```diff
@@ -89,19 +89,19 @@ The following tables contain the exact lifecycle for each published version of A
...
-| 0.26             | Nov 23, 2021     | May 2022                |
-| 0.27             | Dec 21, 2021     | June 2022               |
-| 0.28             | Feb 15, 2022     | February 2023           |
-| 0.29             | June 1, 2022     | December 2022           |
+| Software Version | Release Date    | End of Maintenance Date |
+| ---------------- | --------------- | ----------------------- |
+| 0.25             | May 11, 2021    | December 2022*         |
+| 0.28             | Feb 15, 2022    | February 2023           |
+| 0.29             | June 1, 2022    | December 2022           |
+| 0.30             | August 29, 2022 | February 2022           |
```
Comment (jwitz) [inline software_versioned_docs/version-0.25/release-lifecycle-policy.md:97]:
```suggestion
| 0.30             | August 29, 2022 | August 2023           |
```

File: software_versioned_docs/version-0.28/release-lifecycle-policy.md [hunk 8]
```diff
@@ -89,19 +89,19 @@ The following tables contain the exact lifecycle for each published version of A
...
-| 0.26             | Nov 23, 2021     | May 2022                |
-| 0.27             | Dec 21, 2021     | June 2022               |
-| 0.28             | Feb 15, 2022     | February 2023           |
-| 0.29             | June 1, 2022     | December 2022           |
+| Software Version | Release Date    | End of Maintenance Date |
+| ---------------- | --------------- | ----------------------- |
+| 0.25             | May 11, 2021    | December 2022*         |
+| 0.28             | Feb 15, 2022    | February 2023           |
+| 0.29             | June 1, 2022    | December 2022           |
+| 0.30             | August 29, 2022 | February 2022           |
```
Comment (jwitz) [inline software_versioned_docs/version-0.28/release-lifecycle-policy.md:97]:
```suggestion
| 0.30             | August 29, 2022 | August 2023 |
```

File: software_versioned_docs/version-0.29/release-lifecycle-policy.md [hunk 9]
```diff
@@ -87,17 +87,19 @@ The following tables contain the exact lifecycle for each published version of A
...
-| ---------------- | ---------------- | ----------------------- |
-| 0.25             | May 11, 2021     | December 2022*          |
-| 0.28             | Feb 15, 2022     | February 2023           |
-| 0.29             | June 1, 2022     | December 2022           |
+| Software Version | Release Date    | End of Maintenance Date |
+| ---------------- | --------------- | ----------------------- |
+| 0.25             | May 11, 2021    | December 2022*         |
+| 0.28             | Feb 15, 2022    | February 2023           |
+| 0.29             | June 1, 2022    | December 2022           |
+| 0.30             | August 29, 2022 | February 2022           |
```
Comment (jwitz) [inline software_versioned_docs/version-0.29/release-lifecycle-policy.md:95]:
```suggestion
| 0.30             | August 29, 2022 | August 2023           |
```

**Pull request merged**